source/agora/test/Base.d

```diff
@@ -131,7 +131,7 @@ public class TestAPIManager
     public RemoteAPI!TestAPI[PublicKey] apis;
 
     /// Initialize a new node
-    protected void createNewNode (PublicKey address, Config conf)
+    public void createNewNode (PublicKey address, Config conf)
```

Comment:
It seems `protected` is enough, according to the next commit

Reply:
`createNewNode` needs to be public as it's accessed from the `Base` module, and DMD has a bug: https://issues.dlang.org/show_bug.cgi?id=20002